channel.go

```diff
@@ -124,10 +124,6 @@ func (ch *channel) recv() (messageHeader, []byte, error) {
 	}
 
 	if mh.Length > uint32(messageLengthMax) {
-		if _, err := ch.br.Discard(int(mh.Length)); err != nil {
```

Comment:
The data still needs to be discarded since the message is not used

Reply:
This length field is probably not true, there is not so much data.